src/components/Form/Form.styled.jsx

```diff
@@ -61,7 +77,23 @@ export const FormButtonSecondary = styled.button`
     box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   }
 
-  @media (min-width: 1025px) {
-    width: 200px;
+  @media (min-width: 1200px) {
+    width: 230px;
+  }
+
+  @media (max-width: 1025px) {
+    width: 230px;
+  }
+
+  @media (max-width: 768px) {
+    width: 292px;
+  }
+
+  @media (max-width: 480px) {
+    width: 293px;
+  }
+
+  @media (max-width: 280px) {
+    width: 203px;
   }
 `
```

Comment:
Please check if exists a better solution for this problem to reduce media queries quantity